spinnaker-monitoring-third-party/third_party/newrelic/install.sh

```diff
@@ -0,0 +1,2 @@
+echo "Hello world. This is a stub of an install script for New Relic."
+pip install newrelic-telemetry-sdk
```

Comment:
Is this still a stub?  Or does `pip install` actually do the needful?  😄

Reply:
it's not required, `newrelic-telemetry-sdk` is in the `requirements.txt` file as well. Not sure how important this script actually is, I could probably just remove it.

Reply:
https://github.com/spinnaker/spinnaker-monitoring/search?q=install.sh&unscoped_q=install.sh

Doesn't look necessary. Let's get rid of it!